### DISCUSSION TITLE
2010/12/08/%E8%BE%9B%E8%8B%A6%E7%9A%84%E8%B1%86%E8%B1%86%E5%A6%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#305] [Announcements]
# 2010/12/08/%E8%BE%9B%E8%8B%A6%E7%9A%84%E8%B1%86%E8%B1%86%E5%A6%88

豆豆妈为了生豆豆可是遭了不少罪。怀孕期间豆豆就不老实，经常把妈妈肚子踢歪。但比起分娩以及之后的几天，这都不算什么。

https://blog.qizhen.xyz/2010/12/08/%E8%BE%9B%E8%8B%A6%E7%9A%84%E8%B1%86%E8%B1%86%E5%A6%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

yage
2011年04月25日 15:16 编辑
impressively, we share this feeling

回复
